enclave_manager/Dockerfile

```diff
@@ -216,28 +289,12 @@ RUN mkdir -p build \
   && cmake .. \
   && make
 
-WORKDIR /project/avalon/common/python
-
-RUN make && make install
-
 WORKDIR /project/avalon/common/crypto_utils
 
 RUN make && make install
 
-WORKDIR /project/avalon/sdk
-
-RUN make && make install
-
 WORKDIR /project/avalon/enclave_manager
 
 RUN echo "Build and Install avalon enclave manager" \
   && make && make install
 
-FROM build_image as final_image
```

Comment:
We can keep these lines to remove python-pip3 from final image as it'll reduce the docker image size

Reply:
It wouldn't because build_image would be added as a layer in final_image. So, final_image size will be sum of build_image size + whatever extra we do inside final_image.

Reply:
I mean can we remove python3-pip at end from build image

Reply:
It wouldn't help because python3-pip installation happens in different layer in build_image. So, adding command to remove python3-pip at the end of build_image wouldn't reduce size.

Reply:
Right. So later when we add the final image build step by copying required files from build stage we can address it then in a separate PR

Reply:
yes